crates/ide-completion/src/render/literal.rs

```diff
@@ -50,9 +50,15 @@ fn render(
     path: Option<hir::ModPath>,
 ) -> Option<Builder> {
     let db = completion.db;
-    let kind = thing.kind(db);
-    let has_call_parens =
-        matches!(completion.path_context(), Some(PathCompletionCtx { has_call_parens: true, .. }));
+    let mut kind = thing.kind(db);
+    let should_add_parens = match completion.path_context() {
+        Some(PathCompletionCtx { has_call_parens: true, .. }) => false,
+        Some(PathCompletionCtx { kind: PathKind::Use | PathKind::Type { .. }, .. }) => {
+            cov_mark::hit!(no_parens_in_use_item);
+            false
+        }
+        _ => true,
+    };
```

Comment:
This is similar to https://github.com/rust-lang/rust-analyzer/blob/995a17fbd9f7cdedfdafec946b1d9e44eff448b4/crates/ide-completion/src/render/function.rs#L201-L210 but that one is limited `has_call_parens` to `PathKind::Expr`